portal-api/api-test-setup.jest.config.js

```diff
@@ -13,7 +13,7 @@ mockFiles.forEach((mockFile) => {
   jest.mock(mockFile);
 });
 
-console.log(`MOCKED FILES: \n${mockFiles.join('\n')}`);
+// console.log(`MOCKED FILES: \n${mockFiles.join('\n')}`);
```

Comment:
I think we could just remove these console.logs?

Reply:
console.log removed